merge_test.go

```diff
@@ -1767,7 +1886,7 @@ func TestMergeArray(t *testing.T) {
 				So(err, ShouldBeNil)
 			})
 
-			Convey("throw an error when delete point cannot be found", func() {
+			Convey("don't throw an error when delete point cannot be found", func() {
```

Comment:
Can you add tests to the PR for (admittedly pathological) cases where a name includes an asterisk?  Or a dot?